### DISCUSSION TITLE
How to adjust the margin/padding of markdown?

### DISCUSSION THREAD

**Discussion by yccheok** [gonzalezreal/swift-markdown-ui#362] [Q&A] [answered]
Thank you for creating such a fantastic library!

I noticed that the markdown examples include margins/padding on the leading and trailing edges. Could you please let me know how to remove or adjust them?

![Screenshot 2024-11-21 at 6 23 23 AM](https://github.com/user-attachments/assets/45599196-c75c-4e22-8945-809c7a277a84)

I’m currently integrating the library into a UIKit app and learning how to customize it to match the app's theme. Any guidance would be greatly appreciated.

```
import SwiftUI
import MarkdownUI

extension Theme {
    static let melonoteTheme = Theme()
        .heading1 { configuration in
            configuration.label
                .markdownTextStyle {
                    FontFamily(.system())
                    FontWeight(.bold)
                    FontSize(30)
                    ForegroundColor(Color(UIColor.tintColor))
                }
        }
}

struct MarkdownView: View {
    let markdownString: String


    var body: some View {
        VStack(alignment: .leading) {
            Markdown {
                markdownString
            }
            .markdownTheme(.melonoteTheme)
            Spacer()
        }
    }
}
```

**Comment by gonzalezreal** [comment, marked as answer]
Hey @yccheok,
The `Markdown` view does not add any padding to the contents. That padding is coming from somewhere else, maybe the theme you are using?

**Reply by yccheok** [reply]
Hi @gonzalezreal,

I believe I might have made a mistake elsewhere, as the issue disappeared after experimenting with the built-in GitHub theme.

Thank you for creating this amazing library! I previously used the `Down` library in my `UIKit` app. Since `Down` relies on attributed strings for presentation, it has its limitations. I really appreciate that your library uses native `SwiftUI` components for rendering, which offers much more flexibility.

I spent some time figuring out how to use your wonderful library in a `UIKit` app and shared my solution here: [GitHub Comment](https://github.com/gonzalezreal/swift-markdown-ui/issues/315#issuecomment-2491241514). I hope it benefits the community.

By the way, could you clarify if the following behavior is expected? When I long-press on a text block, I see a "Copy" menu item but no text selection, even though `.textSelection(.enabled)` is turned on.

Thank you 🙏

![Screenshot 2024-11-21 at 10 00 39 PM](https://github.com/user-attachments/assets/592db2b9-a2f5-4bb2-8865-27570f5d7473)
